### ISSUE TITLE
conflict with mongoid

### ISSUE THREAD

**Issue by MrSena** [kernow/shortcode#38]
conflict with gem 'mongoid', '~> 5.0.0'

For your example has error
```
NoMethodError: undefined method `size' for :quote:Symbol
```
No error if use gem 'mongoid', '< 4.0.0'

**Comment by MrSena** [comment]
Mongoid has code
```
class Symbol
  remove_method :size if instance_methods.include? :size # temporal fix for ruby 1.9
end
```
I patch my app with 
```
class Symbol
  def size
    to_s.length
  end
end
```
This resolve problem

**Comment by kernow** [comment]
@MrSena this is an issue to take up with Mongoid which is removing a method
